src/components/StorageView.jsx

```diff
@@ -36,7 +36,9 @@ class StorageView extends Component {
         <h3 className={viewStyles['set-view-subtitle']}>
           {t('StorageView.storage_title')}
         </h3>
-        {isFetching && <Loading />}
```

Comment:
I don't see this Loading component being removed, we still needed it?

Reply:
It has been removed, check again. :)

Reply:
OK my bad 👍